spatial-tracking-explainer.md

```diff
@@ -364,19 +366,19 @@ xrReferenceSpace.addEventListener('reset', xrReferenceSpaceEvent => {
   // time to reset that.
   resetYawTransform(transform);
 
-  // For an app using the XRBoundedReferenceSpace, this would be a perfect time to
+  // For an app using a bounded reference space, this would be a perfect time to
   // re-layout content intended to be reachable within the bounds
   createBoundsMesh(transform);
 });
 ```
 
 Example reasons `onreset` may fire:
-* For an `XRStationaryReferenceSpace`, some XR systems have a mechanism for allowing the user to reset which direction is "forward" or re-center the scene's origin at their current location.
-* For an `XRBoundedReferenceSpace`, a user steps outside the bounds of a "known" playspace and enters a different "known" playspace
-* For an `XRUnboundedReferenceSpace`, an inside-out based tracking system is temporarily unable to locate the user (e.g. due to poor lighting conditions in a dark hallway) and then recovers tracking in a new map fragment that cannot be related to the previous map fragment
-* For an `XRUnboundedReferenceSpace`, when the user has travelled far enough from the origin of the reference space that floating point error would become problematic
+* For an `eye-level` or `floor-level` reference space, some XR systems have a mechanism for allowing the user to reset which direction is "forward" or re-center the scene's origin at their current location.
+* For a `bounded` reference space, a user steps outside the bounds of a "known" playspace and enters a different "known" playspace
+* For a `bounded` reference space, an inside-out based tracking system is temporarily unable to locate the user (e.g. due to poor lighting conditions in a dark hallway) and then recovers tracking in a new map fragment that cannot be related to the previous map fragment
+* For an `unbounded` reference space, when the user has travelled far enough from the origin of the reference space that floating point error would become problematic
```

Comment:
Can we use `{{XRReferenceSpaceType/foo}}` for all of these?

Reply:
This is explainer text rather than spec text, so I don't believe we can use those references here.